index.bs

```diff
@@ -71,8 +74,32 @@ Examples {#examples}
 Security and Privacy Considerations {#security-and-privacy}
 ===================================
 
-There are no specific security and privacy considerations
-beyond those described in the Generic Sensor API [[!GENERIC-SENSOR]].
+Ambient Light Sensor provides information about lighting conditions near
+the device environment. Potential privacy risks include:
+
+  - Information leaks about the user’s surrounding and work habits.
+  - Profiling. Readout from Ambient Light Sensor can potentially induce
+    information leaks about the user’s interests, web use and work
```

Comment:
I'll let @lknik comment on this text (a copy-paste from https://github.com/w3c/ambient-light/issues/13#issue-174271517 with some very minor copyediting).